### ISSUE TITLE
localhost

### ISSUE THREAD

**Issue by ghost** [p-acs/ethereum-android-lib#9]
it is possible connect with localhost and test it  ?

**Comment by Rchik** [comment]
Yes. I was able to test both desktop and mobile android client with the same ethereum instance, sending money and invoking contract methods. You will need vertigo/ethereum and pacs/ethereum-secure-proxy docker images and some extra work with your hands.
